.github/workflows/code-quality.yml

```
@@ -0,0 +1,171 @@
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Run secret scanning
+        run: |
+          echo "🔐 Scanning for potential secrets..."
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)